Assignment-2/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    
+    <title>Flawless Skincare</title>
+</head>
+<body>
+    <div class="color-example"><h1>Flawless Glowing Skincare Routine </h1></div>
+   <br>
+
+
+    
+
+    <img src="https://ouishave.com/wp-content/uploads/2021/10/Black-Skincare-Routine.jpeg" alt="image" class="center">
+
+<br>
+
+
+    <h2>Welcome to the world of skincare! This blog is based around all things skincare. Good skin care is not just about looking good, it's about feeling good too. Your skin is a refection of your inner health. I hope you enjoy these tips, must haves & product suggestions! </h2>
+
+    
+
+    <ol>
+        <p>Must Have Items</p>
+        <dl>A Great Cleanser</dl>
+        <dl>Toner w/ PHA, BHA</dl>
+        <dl>Day/Night Moisturizer</dl>
+        <dl>Hyaluronic Acid 2% + B5</dl>
+        <dl>SPF 30+</dl>
+    </ol>
+    <ol>
+        <p>Skincare Product Order</p>
+        <dl>Cleanser</dl>
+        <dl>Toner</dl>
+        <dl>Spot Treatment</dl>
+        <dl>Serums</dl>
+        <dl>Retinol</dl>
+        <dl>Eye Cream</dl>
+        <dl>Moisturizer</dl>
+        <dl>Face Oils</dl>
+    </ol>
+    <ol>
+    <p>My Favorite Products</p>
+    <dl>Fenty</dl>
+    <dl>Glow Recipe</dl>
+    <dl>The Ordinary</dl>
+    <dl>Laneige</dl>
+</ol>
+
+<img src="https://i.ibb.co/JqWhJVG/skincare.jpg" alt="image" class="center">
```

Comment:
Alt attributes should be more descriptive for accessibility reasons. a better alt attribute than image would be 'skincare products' for example